db/migrate/20210202052337_migrate_variant_unit_values.rb

```diff
@@ -0,0 +1,13 @@
+class MigrateVariantUnitValues < ActiveRecord::Migration
+  def up
+    Spree::Variant.all.select { |v|
+      v.unit_value.nil? && v.product&.variant_unit == "items"
+    }.each do |variant|
```

Comment:
This will literally blow up the server because we are loading heaps of records into memory. We should rather query the ones we care about by joining variants and products together.